### PR TITLE
fixes #3951 chore(nimbus): validate v6 serializer with schema v1.1.0

### LINKED ISSUE #3951: Update schema validation for serializer v6 to 1.1.0
Now that the schema is published with `additionalProperties` allowed, we can restore validation in the v6 serializer.

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/EXP-568)
┆Issue Number: EXP-568

### PR DESCRIPTION
**Pull request by k88hudson**
Because

* v1.1.0 of nimbus_shared adds support for additionalProperties

This commit

* Upgrade nimbus_shared and restore validation for v6 serializer

### CODE REVIEW

File: app/pyproject.toml [hunk 1]
```diff
@@ -53,7 +53,7 @@ isort = "5.6.4"
 google-cloud-storage = "1.32.0"
 django-storages = "1.10.1"
 graphene-django = "^2.13.0"
-mozilla-nimbus-shared = "^1.0.0"
+mozilla-nimbus-shared = "^1.1.0"
```
Comment (pdehaan) [inline app/pyproject.toml:56]: Looks like experimenter/app/experimenter/legacy-ui/rapid/package.json is still on v0.0.9. Not sure if that creates any potential conflicts.

https://github.com/mozilla/experimenter/blob/6e66462cbf57a721018b8f86251ad8a9ee0b1a7e/app/experimenter/legacy-ui/rapid/package.json#L7

Comment (jaredlockhart) [reply app/pyproject.toml:56]: No it needs that version, so that's correct.

Comment (jaredlockhart) [reply app/pyproject.toml:56]: But thnx for checking 👍

**Review by jaredlockhart — APPROVED** [review]
Very nice ty 👍

**Pull request merged**